### PR TITLE
Progress page

### PR DESCRIPTION
**Pull request by larsmennen**
Adds a page detailing progress in the last 200 years, challenges and what people can do. It'll be at altruisto.com/progress.

@kovsky0 main changes since the version on gitlab:

1. Merged into this repo, at `/progress`
2. Bumped a couple of dependencies to make this work on Mac OS Big Sur (particularly node-sass and webpack). There are some small additional changes in the main website's codebase in that commit that were needed to make this work.
3. The page is now responsive. For mobile viewports in landscape mode, we show an overlay telling the user to rotate their device. For mobile viewports in portrait mode, the fullpage.js scrolling works up to viewport heights of 666px (iPhones >5 have larger screens). Below that it becomes normal scrolling.
4. Updated the text and added the missing bits on the last few sections.
5. Added a couple of backgrounds.
6. I split the webpack config into two pipelines (one config): one generates `index.js` for the existing pages, and one generates `progress.js` for this page.

### CODE REVIEW

**Comment by kovsky0** [comment]
1. `/progress-of-humanity` maybe

File: website/custom/index/assets/js/progress.js [hunk 1]
```diff
@@ -0,0 +1,377 @@
+import './jquery.svg.min';
+import './jquery.svganim.min';
+
+$(document).ready(function() {
+
+  // Graphs
+  const width = 0.75; // in vw
```
Comment (kovsky0) [inline website/custom/index/assets/js/progress.js:7]: units in var name

File: website/custom/index/assets/js/progress.js [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+      'type': 'circle',
+      'stroke': {
+        'width': 0,
+        'color': '#000000'
+      },
+      'polygon': {
+        'nb_sides': 5
+      },
+      'image': {
+        'src': 'img/github.svg',
```
Comment (kovsky0) [inline website/custom/index/assets/js/progress.js:290]: this is something stranger

File: website/custom/index/progress.html [hunk 3]
```diff
@@ -0,0 +1,608 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml">
+
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <title>The progress humanity made in the last 200 years</title>
+    <meta name="author" content="Daniel Wyrzykowski, Michal Trzesimiech, Marcin Wosinek, Adam Bac" />
```
Comment (kovsky0) [inline website/custom/index/progress.html:7]: You can put yourself here

File: website/custom/index/progress.html [hunk 4]
```diff
@@ -0,0 +1,608 @@
...
+    <meta name="description" content="See the data showing the progress humanity made in the last two centuries." />
+    <meta name="keywords"  content="data, progress, infographic" />
+    <meta name="Resource-type" content="Document" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1">
+
+
+    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/fullPage.js/2.9.7/jquery.fullpage.css" />
```
Comment (kovsky0) [inline website/custom/index/progress.html:14]: I would serve it locally

File: website/custom/index/progress.html [hunk 5]
```diff
@@ -0,0 +1,608 @@
...
+
+    <!--[if IE]>
+    <script type="text/javascript">
+        var console = { log: function() {} };
+    </script>
+    <![endif]-->
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
+    <script src="https://d3js.org/d3.v5.min.js"></script>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/fullPage.js/2.9.7/jquery.fullpage.min.js"></script>
```
Comment (kovsky0) [inline website/custom/index/progress.html:420]: let's try to move it to serve locally

**Comment by kovsky0** [comment]
The line of a graph doesn't draw to the end

**Comment by kovsky0** [comment]
Don't hide the content of last slide when we show footer

**Comment by kovsky0** [comment]
Share button - let's update it if we change the path of the page

**Comment by kovsky0** [comment]
If you start in landscape - we need to update variables to recompute when mode changes

**Comment by kovsky0** [comment]
Transparency to background behind text (so that we see photo behind it)

**Comment by kovsky0** [comment]
Last slide - move background so that face is visible

**Comment by larsmennen** [comment]
Add husky

**Comment by kovsky0** [comment]
Please also add analytics:
```
<script type="text/javascript">
      ;(function (i, s, o, g, r, a, m) {
        i["GoogleAnalyticsObject"] = r
        ;(i[r] =
          i[r] ||
          function () {
            ;(i[r].q = i[r].q || []).push(arguments)
          }),
          (i[r].l = 1 * new Date())
        ;(a = s.createElement(o)), (m = s.getElementsByTagName(o)[0])
        a.async = 1
        a.src = g
        m.parentNode.insertBefore(a, m)
      })(window, document, "script", "https://www.google-analytics.com/analytics.js", "ga")

      ga("create", "UA-79596452-2", "auto")
      ga("send", "pageview")
    </script>
```

**Comment by larsmennen** [comment]
@kovsky0 do you want to take a final look? Think all comments are addressed now. Good from my point of view.

**Pull request merged**